.github/workflows/codeql.yml

```
@@ -51,3 +51,34 @@ jobs:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v4
+        with:
+          languages: swift
+          build-mode: manual
+
+      - name: Type-check Swift sources for CodeQL extraction
+        timeout-minutes: 5
+          run: >-
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)